src/features/projectsV2/ProjectsMap/microComponents/SatelliteLayer.tsx

```diff
@@ -0,0 +1,24 @@
+import { useMemo } from 'react';
+import { Layer, Source } from 'react-map-gl-v7';
+
```

Comment:
Imports should be from react-map-gl-v7/maplibre instead. Otherwise we're mixing mapbox and maplibre, which could be problematic.